sklearn/datasets/base.py

```diff
@@ -263,6 +269,9 @@ def load_iris():
         'target_names', the meaning of the labels, 'feature_names', the
         meaning of the features, and 'DESCR', the
         full description of the dataset.
+    
+    (data,target) : tuple if return_X_y == true
```

Comment:
try upgrading your sphinx (I'm on 1.4.5). Otherwise, our systems look pretty similar
```
Darwin-15.5.0-x86_64-i386-64bit
('Python', '2.7.12 (default, Jun 29 2016, 14:05:02) \n[GCC 4.2.1 Compatible Apple LLVM 7.3.0 (clang-703.0.31)]')
('NumPy', '1.11.1')
('SciPy', '0.17.1')
('Scikit-Learn', '0.18.dev0')
```